llvm_mode/afl-clang-fast.c

```diff
@@ -353,7 +357,9 @@ int main(int argc, char** argv) {
   }
 
 
+#ifndef __ANDROID__
   find_obj(argv[0]);
```

Comment:
also why does this fail?

Reply:
Android doesn't have afl-llvm-rt.o generated, it should be afl-llvm-rt.o.o.
Actually due to the different build system other than normal make, there is no need to check afl-llvm-rt.o in Android.